rrfs-test/validated_yamls/templates/obtype_config/msonet_winds_288.yaml

```diff
@@ -0,0 +1,476 @@
"LKBNV","STHNG","MNRCM","CLRAH","LANGL","BFFLS","BONHM","MARLN","STBRG","GENEA","TOMBL","KNWAT","AUROH","WLMSV","NEWTN","JRMYN","BRDMN","FMYSN","MDSN1","VCHMN","DBNNW","STHNR","CRNX1","RCHWR","SNDI0","BLLVS","MCGVL","EGLRV","CMPBL","JZLTN","LEROY","BNTSL","PRRRY","NPRVD","CHRHU","CLWST","EGLNS","BRNSW","HILMR","LNXHS","CDDML","EURKB","SLNSA","PRTBY","FRSTB","FRDMY","SBS02","MNCNS","ELWRT","FMNTN","STLFC","EDNBR","TRFLG","BGLSB","SPRCT","HRHES","WFORT","DMOND","LIZTN","YKSHR","CHALS","SWFTW","WLLBG","AQLLI","GRNS1","NSTHH","HDLRD","PRVON","BFFLM","KWIES","MRCRS","PLMTT","PGHKA","JSTON","FRTDF","NAPA1","LCKPT","MERDN","POWHA","WNTZV","TOPKA","WTTRV","PLMTS","RSFRD","CAPCL","FTBCM","DISPT","NEWWM","MRAMR","APXNC","MIDBR","SPIGN","LEBON","QUAKT","FPDPT","SLLVN","TRYBU","JMS02","PHNSM","CESSC","HRSHM","CLFCO","POETT","BTHLS","BRNTW","MKSCR","ACMRS","BRUSW","NAMSS","WWLPG","STRLI","HGBRD","RCHST","NPLSL","CHRRS","FRMDD","CHRVL","PCHUT","KLEIN","MTENT","ABGTN","GUILF","MIAM2","RLGH1","TCMMF","WZVNT","MSHWL","BOLVR","OKFLD","GRNDR","LTHM1","BEFFL","PRNCS","BDFDV","WRWRG","WNTRE","FREDT","MKERK","NWTON","RCHDL","RSNGS","WAVRL","NSTSU","CRNNB","PWTCK","ESTFR","NWDST","WLDNP","DNSPS","BRDJW","SLDNR","ASHBR","NRTHU","MLTNL","NVADA","RSNRS","DOLPH","ELCTV","GBSNA","SAVTN","MNTVN","BTHMY","SMTHF","LESSF","MRCIL","CRSPO","SEAWR","SNJSE","NRTHR","A0031","LXNPP","CLRPH","MATLD","GSOCM","ASHSH","HSLND","PCBCH","WHTHL","MINLA","SRCHL","PRRV2","SPRHE","SPGDL","WWLPW","HLMSB","DNSTJ","CRTLS","SVGRD","AHSKE","MNSMF","VCLRV","ABRNS","BLAIN","LNNLN","BRSTW","FRANK","WKFST","WSWEC","BSO02","PRCSD","LSBES","DLVL3","CAPCO","ULYSS","BKFLD","EHAVN","KSSMS","OKLYV","MSCWL","JSPER","STHDY","WLMNE","EVJHA","DLLWY","DCECX","AKRNB","OSCLA","LANDO","DANVL","PTSTW","SYRCS","SLMCC","KNGWL","GRDNC","BRCKP","UNVPL","MARNO","CLMSN","RALGH","MNMNV","CHRDN","FRDBG","HMDEN","MRMAR","LAWNC","FRMNJ","DDLEY","WESSN","TACPE","WHNT2","DOYLT","MCHVL","WTRTN","ESTRR","MDLJR","NWTN1","PKTNN","JKKSN","NWIVN","PRRFR","MNNDR","BFMMA","OCCTY","OLYWB","ANDRW","CMBDC","KETUM","BNNYL","WLLWP","MSHIM","DIMCK","HFFMN","ALFRD","FLMDT","A0082","CRNLS","SEEKK","TKKWL","HGHSV","JSPRH","RSFHM","YANKE","MDGRD","WSTGR","PUALP","KNTWA","TACMA","HIGDN","OLDWS","DTRTV","RTTMN","WALLR","HESPE","WTNHT","HNVRT","ASTRA","FRKNN","MDBJH","BSO07","SARVR","ESTHS","CNTPL","PACSC","OKHRB","CAMAS","BRWLL","CBANY","MNTPL","BTHHA","DACSC","OFLCS","ROCHE","OZRDC","THMLM","ATOWN","GGAPL","EVADL","KUNTZ","MINRL","WNSKT","SLTHL","DNDNN","LGNTN","HLLRT","RLGHD","LXNGT","BRESC","RYREV","RBSNC","FWOTH","TCMAW","ALLNN","ATTLL","BOYER","UNVDE","JIMLW","CHNTL","WLCTT","LKWDA","CRMEL","SNYVL","CUMBR","OLEAN","SPGVL","PIRSN","SPNDR","HUTSV","NEWNJ","CLLMN","SHPBG","ETNVL","OTHLL","MOSCW","LXNGS","NTCHZ","GBFHP","NBUOH","ALTMN","MANSF","MDDON","STHLD","CENSQ","CHTTN","SBURY","MCHNC","VNCMB","NRTBY","DFBS1","PHLFF","DLND4","SMSTX","BBRNC","ALLCS","CHATT","HAMBR","FLKVL","GULF2","ELBAC","GBORO","WSTPA","MRRRJ","SLDAD","LYNJS","WRWNY","NORWH"]
```

Comment:
My gosh on this

Reply:
tag @hu5970 to see this.

Reply:
If there is not function or filter to read the mesonet uselist file for now, we can add this as a item for the future development.